src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,22 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { Todo } from '../../types/types';
+import usersFromServer from '../../api/users';
+import { UserInfo } from '../UserInfo';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const foundUser = usersFromServer.find(u => u.id === todo.userId);
```

Comment:
Take another look at this variable 'u'. Perhaps it would be clearer if it had a more specific name, such as 'person', 'userUnit' or something similar.